spec/trim_decorator_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../trim_decorator'
+
+describe TrimmerDecorator do
+  before :each do
+    @teacher = Teacher.new(55, 'Science', 'James Smith Jr.')
+  end
+  context 'test trimmer' do
+    it 'should return a name trimmed' do
+      @trimmed = TrimmerDecorator.new(@teacher)
+      expect(@trimmed.correct_name).to eql 'James Smit'
+    end
+  end
+end
```

Comment:
- [ ] [REQUIRED] Kindly ensure that the test for the `TrimmerDecorator` class passes as it currently throws an error that does not identify the teacher constant.
![Screenshot from 2022-07-28 00-17-21](https://user-images.githubusercontent.com/58692590/181388954-fc77469f-1b8f-46a6-ad8b-301d092e470f.png)

Reply:
When running this test alone, it yields the error because of missing requires. I'll fix it